tableau-databricks/manifest.xml

```diff
@@ -147,7 +147,7 @@ limitations under the License.
         <customization name='CAP_ODBC_METADATA_SUPPRESS_SQLCOLUMNS_API' value='no' />
         <customization name='CAP_ODBC_METADATA_SUPPRESS_SELECT_STAR' value='yes' />
         <customization name='CAP_ODBC_SUPPRESS_CATALOG_NAME' value='no'/>
-        <customization name='CAP_QUERY_TOP_0_METADATA' value='no' />
+        <customization name='CAP_QUERY_TOP_0_METADATA' value='yes' />
```

Comment:
I don't think we should enable metadata using LIMIT queries because they need to go through analysis and overall they are slower than the Thrift native RPCs.